test/test_prims.py

```diff
@@ -752,6 +759,34 @@ def fn1(x):
             )
             self.assertFalse(includes_aten_to_copy)
 
+    @ops([op for op in op_db if op.supports_varargs], allowed_dtypes=(torch.float,))
```

Comment:
The dtype-related parameters in the `@ops` decorator are a little wonky because there's both `allowed_dtypes` and `dtypes`. See 

https://github.com/pytorch/pytorch/blob/15435325eb6c561f8a126b5c4e48ac57c064a6fa/torch/testing/_internal/common_device_type.py#L709

In this case `dtypes=OpDtypes.any_one` might better capture the desired semantics (although almost all operators support the `float32` datatype, not all do)